sklearn/linear_model/_ridge.py

```diff
@@ -1704,6 +1741,13 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
         below). This flag is only compatible with ``cv=None`` (i.e. using
         Generalized Cross-Validation).
 
+    alpha_per_target : boolean, default=False
```

Comment:
```suggestion
    alpha_per_target : bool, default=False
```
See the [guidelines for writing documentation](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation)